test/sequential/bootstrap.checkActivatePool.test.ts

```diff
@@ -0,0 +1,188 @@
+  getLiquidityAssetId,
+  claimAndActivateBootstrap,
+} from "../../utils/tx";
+import { EventResult, ExtrinsicResult } from "../../utils/eventListeners";
+import { Keyring } from "@polkadot/api";
+import { User } from "../../utils/User";
+import {
+  getEnvironmentRequiredVars,
+  waitForBootstrapStatus,
+} from "../../utils/utils";
+import {
+  getEventResultFromMangataTx,
+  sudoIssueAsset,
+} from "../../utils/txHandler";
+import { MGA_ASSET_ID } from "../../utils/Constants";
+import { BN, toBN } from "@mangata-finance/sdk";
+import { Assets } from "../../utils/Assets";
+import { Sudo } from "../../utils/sudo";
+import { setupApi, setupUsers } from "../../utils/setup";
+
+jest.spyOn(console, "log").mockImplementation(jest.fn());
+jest.spyOn(console, "error").mockImplementation(jest.fn());
+jest.setTimeout(3500000);
+process.env.NODE_ENV = "test";
+
+let testUser1: User;
+let sudo: User;
+let keyring: Keyring;
+let bootstrapPhase: any;
+let bootstrapCurrency: any;
+let bootstrapPool: any;
+let eventResponse: EventResult;
+let bootstrapPoolBalance: BN;
+
+const { sudo: sudoUserName } = getEnvironmentRequiredVars();
+const waitingPeriod = 15;
+const bootstrapPeriod = 30;
+const whitelistPeriod = 10;
+const bootstrapAmount = new BN(10000000000);
+
+async function checkPossibilityCreatingPool(tokenA: any, tokenB: any) {
+  const creatingPool = await (
+    await getMangataInstance()
+  ).createPool(
+    testUser1.keyRingPair,
+    tokenA.toString(),
+    bootstrapAmount,
+    tokenB.toString(),
+    bootstrapAmount
+  );
+  eventResponse = getEventResultFromMangataTx(creatingPool);
+  expect(eventResponse.state).toEqual(ExtrinsicResult.ExtrinsicFailed);
+  expect(eventResponse.data).toContain("DisallowedPool");
+}
+
+beforeAll(async () => {
+  try {
+    getApi();
+  } catch (e) {
+    await initApi();
+  }
+
+  keyring = new Keyring({ type: "sr25519" });
+
+  // setup users
+  sudo = new User(keyring, sudoUserName);
+  testUser1 = new User(keyring);
+
+  [testUser1] = setupUsers();
+
+  const bootstrapCurrencyIssue = await sudoIssueAsset(
+    sudo.keyRingPair,
+    toBN("1", 20),
+    sudo.keyRingPair.address
+  );
+  const bootstrapEventResult = await getEventResultFromMangataTx(
+    bootstrapCurrencyIssue,
+    ["tokens", "Issued", sudo.keyRingPair.address]
+  );
+  const bootstrapAssetId = bootstrapEventResult.data[0].split(",").join("");
+  bootstrapCurrency = new BN(bootstrapAssetId);
+});
+
+test("bootstrap - Check that we can not create a pool for the bootstrap token after bootstrap was declared", async () => {
+  const api = getApi();
+  await setupApi();
+
+  // check that system is ready to bootstrap
+  bootstrapPhase = await api.query.bootstrap.phase();
+  if (bootstrapPhase.toString() === "Finished") {
+    const bootstrapFinalize = await finalizeBootstrap(sudo);
+    eventResponse = getEventResultFromMangataTx(bootstrapFinalize);
+    expect(eventResponse.state).toEqual(ExtrinsicResult.ExtrinsicSuccess);
+  }
+
```

Comment:
I think you forgot the bootstrap.status === "BeforeStart" expect, that I think is the most important ones

Reply:
I added checking bootstrap.status === "BeforeStart" in L107-108. This is what you meant, right?

Reply:
the most suitable place is on the before all  or before each, as you did in previous tests, but that's fine if its the only test